build-binutils.py

```diff
@@ -137,15 +174,24 @@ def invoke_make(build_folder, install_folder, target):
         ['make', '-s', '-j' + str(multiprocessing.cpu_count()), 'V=0'],
         check=True,
         cwd=build_folder.as_posix())
-    subprocess.run(
-        ['make', '-s', 'prefix=' + install_folder.as_posix(), 'install', 'V=0'],
-        check=True,
-        cwd=build_folder.as_posix())
+    subprocess.run([
+        'make', '-s', 'prefix=' + install_folder.as_posix(), 'install', 'V=0'
+    ],
```

Comment:
did the formatter choose this indentations for `],`?

Reply:
Yes.